src/test/ui/parser/raw/raw-str-too-many-hashes.rs

```diff
@@ -0,0 +1,7 @@
+// ignore-tidy-linelength
```

Comment:
In the PR that introduced this error it was decided not to add this test - https://github.com/rust-lang/rust/pull/50296.

Reply:
Oh, thanks. I'll remove it and leave a comment about that decision somewhere.